.csslintrc

```diff
@@ -0,0 +1,28 @@
+{
+  "adjoining-classes": false,
+  "box-sizing": false,
+  "box-model": false,
+  "bulletproof-font-face": 2,
+  "display-property-grouping": 2,
+  "duplicate-background-images": 2,
+  "duplicate-properties": 2,
+  "empty-rules": 2,
+  "floats": 2,
+  "outline-none": 2,
+  "font-faces": 2,
+  "ids": 2,
+  "import": 2,
+  "important": 2,
+  "known-properties": 2,
+  "qualified-headings": 2,
+  "regex-selectors": 2,
+  "shorthand": 2,
+  "star-property-hack": 2,
+  "text-indent": 2,
+  "unqualified-attributes": false,
+  "underscore-property-hack": 2,
+  "unique-headings": 2,
+  "universal-selector": false,
```

Comment:
:+1:

Using `*` is fine when on it's own for things like box-sizing, but otherwise should be avoided. Shame there's not a rule for that.